src/algebra/order/hom/monoid.lean

```diff
@@ -171,6 +171,20 @@ lemma map_nonpos (ha : a ≤ 0) : f a ≤ 0 := by { rw ←map_zero f, exact orde
 
 end ordered_add_comm_monoid
 
+section ordered_add_comm_group
+variables [ordered_add_comm_group α] [ordered_add_comm_monoid β] [add_monoid_hom_class F α β]
+  {f : F}
```

Comment:
Even if that's an `iff`, I would keep `f` explicit to support applying this lemma to unbundled morphisms (by bundling them on the fly)
```suggestion
  (f : F)
```